src/app/project/project-documents/detail/detail.component.html

```diff
@@ -31,6 +31,14 @@ <h1>
       <h3 for="displayName">Display Name: </h3>
       <p id="displayName">{{document.displayName || '-'}}</p>
     </div>
+    <div class="label-pair">
```

Comment:
Reading the ticket, are we sure we want to display this information yet? If so, I wonder if the general user might not understand the word "hit" and might prefer something like "views"?

Reply:
Yeah, I agree. The ticket kinda has two stories; one for implementing and one for display, but the AC for the ticket is only really about implementation. Having the values visible somewhere makes things easier for testing. I'll update the labels from hits to views for now and we can likely remove these labels when/if we move the display to the table.